src/custom-types.json

```diff
@@ -803,6 +803,14 @@
   "text/x-handlebars-template": {
     "extensions": ["hbs"]
   },
+  "text/x-java-properties": {
+    "compressible": true,
+    "extensions": ["properties"],
+    "sources": [
+      "https://stackoverflow.com/a/28813973",
+      "https://www.digipres.org/formats/mime-types/"
```

Comment:
https://github.com/jshttp/mime-db#contributing

> links to type aggregating sites [...] are not acceptable

Reply:
I think this will be https://trac.edgewall.org/wiki/TracSyntaxColoring and http://wiki.cacert.org/HelpOnParsers

Reply:
Are those valid primary sources?

Reply:
https://trac.edgewall.org/wiki/TracSyntaxColoring does not look like a primary source, as they did not create this type/extension and whatever the spec for it's contents; it is just listing information for how their editor works.

http://wiki.cacert.org/HelpOnParsers does not look like a primary source, as they did not create this type/extension and whatever the spec for it's contents; it is just listing information for how their parser works.

These would be considered secondary sources; they would have gotten that information from some primary location.

Reply:
I can't find any source that can be considered primary. But this is commonly used mime type for `.properties` files

Reply:
Thank you. Without a primary source, we cannot just add it here directly. You should follow the process at https://github.com/jshttp/mime-db#adding-custom-media-types